pkg/handler/rpms.go

```diff
@@ -130,3 +131,39 @@ func (rh *RpmHandler) searchSnapshotRPMs(c echo.Context) error {
 	}
 	return c.JSON(200, resp)
 }
+
+// listSnapshotRpm godoc
+// @Summary      List Snapshot RPMs
+// @ID           listSnapshotRpms
+// @Description  List RPMs in a repository snapshot.
+// @Tags         snapshots
+// @Accept       json
+// @Produce      json
+// @Param		 uuid	path string true "Snapshot ID."
+// @Param		 limit query int false "Number of items to include in response. Use it to control the number of items, particularly when dealing with large datasets. Default value: `100`."
+// @Param		 offset query int false "Starting point for retrieving a subset of results. Determines how many items to skip from the beginning of the result set. Default value:`0`."
+// @Param		 search query string false "Term to filter and retrieve items that match the specified search criteria. Search term can include name."
+// @Success      200 {object} api.RepositoryRpmCollectionResponse
```

Comment:
I think it's because this should be "SnapshotRpmCollectionResponse" ?

Reply:
thanks! updated